xlineapi/src/command.rs

```diff
@@ -508,6 +509,82 @@ impl PbCodec for Command {
     }
 }
 
+/// Get command keys from a Request for conflict check
+pub trait CommandKeys {
+    /// Key ranges
+    fn keys(&self) -> Vec<KeyRange>;
```

Comment:
Is this name conflict with `Command::keys`? `key_ranges` might be a better name.